src/main/java/jenkins/branch/WorkspaceLocatorImpl.java

```diff
@@ -403,8 +406,8 @@ public void onDeleted(Item item) {
             TopLevelItem tli = (TopLevelItem) item;
             Jenkins jenkins = Jenkins.get();
             Computer.threadPoolForRemoting.submit(new CleanupTask(tli, jenkins));
-            for (Node node : jenkins.getNodes()) {
-                Computer.threadPoolForRemoting.submit(new CleanupTask(tli, node));
+            // Starts provisioner Thread which is tasked with starting cleanup Threads
+            new CleanupTaskProvisioner(tli, jenkins.getNodes()).run();
             }
```

Comment:
```suggestion
            new CleanupTaskProvisioner(tli, jenkins.getNodes()).run();
```

Reply:
@kryptag The indentation in the file is all messed up now because you removed the opening brace of the loop but kept the closing one. That's why you had to remove `@Override` from some methods and probably why the build is failing. You need to apply @bitwiseman's suggestion and remove the extra closing brace.